### PR TITLE
New package: overwolf.overwolf version 2.205.0.4

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

### CODE REVIEW

**Review by solomoncyj — COMMENTED** [review]
metaday

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=NVS1.7-3-0-preview-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Overwolf.Overwolf
+PackageVersion: 2.112.1.4
+PackageLocale: en-US
+Publisher: Overwolf
```
Comment (ItzLevvie) [inline manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml:7]:
```suggestion
Publisher: Overwolf Ltd.
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=NVS1.7-3-0-preview-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Overwolf.Overwolf
+PackageVersion: 2.112.1.4
```
Comment (ItzLevvie) [inline manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.installer.yaml:5]:
```suggestion
PackageVersion: 0.116.2.25
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=NVS1.7-3-0-preview-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Overwolf.Overwolf
+PackageVersion: 2.112.1.4
```
Comment (ItzLevvie) [inline manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.116.2.25
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=NVS1.7-3-0-preview-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Overwolf.Overwolf
+PackageVersion: 2.112.1.4
```
Comment (ItzLevvie) [inline manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.yaml:5]:
```suggestion
PackageVersion: 0.116.2.25
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: -Silent
+  SilentWithProgress: -Silent
+UpgradeBehavior: install
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://download.overwolf.com/OverwolfInstaller.exe
+  InstallerSha256: D6261AF9EB474589AF3D3F4DDF8F853395DF7EC5FB3F68F67E3C099E698D9A05
```
Comment (ItzLevvie) [inline manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.installer.yaml:15]:
```suggestion
  InstallerSha256: D6261AF9EB474589AF3D3F4DDF8F853395DF7EC5FB3F68F67E3C099E698D9A05
  ProductCode: Overwolf
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.installer.yaml:5]:
```
Enter a pull request number: https://github.com/microsoft/winget-pkgs/pull/76734

Manifest validation succeeded.
Found Overwolf [Overwolf.Overwolf] Version 2.112.1.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.overwolf.com/OverwolfInstaller.exe
  ██████████████████████████████  1.07 MB / 1.07 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Overwolf
Publisher     : Overwolf Ltd.
Version       : 0.116.2.25
ProductCode   : Overwolf
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Overwolf
Uninstall     : winget uninstall --id "Overwolf"

Name          : Overwolf.Setup.VC100CRTx64.Dist
Publisher     : Overwolf
Version       : 1.0.0
ProductCode   : {EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Uninstall     : winget uninstall --id "{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}"

Press any key to continue . . .
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml:5]:
```
Enter a pull request number: https://github.com/microsoft/winget-pkgs/pull/76734

Manifest validation succeeded.
Found Overwolf [Overwolf.Overwolf] Version 2.112.1.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.overwolf.com/OverwolfInstaller.exe
  ██████████████████████████████  1.07 MB / 1.07 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Overwolf
Publisher     : Overwolf Ltd.
Version       : 0.116.2.25
ProductCode   : Overwolf
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Overwolf
Uninstall     : winget uninstall --id "Overwolf"

Name          : Overwolf.Setup.VC100CRTx64.Dist
Publisher     : Overwolf
Version       : 1.0.0
ProductCode   : {EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Uninstall     : winget uninstall --id "{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}"

Press any key to continue . . .
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.locale.en-US.yaml:7]:
```
Enter a pull request number: https://github.com/microsoft/winget-pkgs/pull/76734

Manifest validation succeeded.
Found Overwolf [Overwolf.Overwolf] Version 2.112.1.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.overwolf.com/OverwolfInstaller.exe
  ██████████████████████████████  1.07 MB / 1.07 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Overwolf
Publisher     : Overwolf Ltd.
Version       : 0.116.2.25
ProductCode   : Overwolf
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Overwolf
Uninstall     : winget uninstall --id "Overwolf"

Name          : Overwolf.Setup.VC100CRTx64.Dist
Publisher     : Overwolf
Version       : 1.0.0
ProductCode   : {EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Uninstall     : winget uninstall --id "{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}"

Press any key to continue . . .
```

File: manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.yaml (hunk 4 above)
Comment (ItzLevvie) [reply manifests/o/Overwolf/Overwolf/2.112.1.4/Overwolf.Overwolf.yaml:5]:
```
Enter a pull request number: https://github.com/microsoft/winget-pkgs/pull/76734

Manifest validation succeeded.
Found Overwolf [Overwolf.Overwolf] Version 2.112.1.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.overwolf.com/OverwolfInstaller.exe
  ██████████████████████████████  1.07 MB / 1.07 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Overwolf
Publisher     : Overwolf Ltd.
Version       : 0.116.2.25
ProductCode   : Overwolf
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Overwolf
Uninstall     : winget uninstall --id "Overwolf"

Name          : Overwolf.Setup.VC100CRTx64.Dist
Publisher     : Overwolf
Version       : 1.0.0
ProductCode   : {EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}
Uninstall     : winget uninstall --id "{EC9D5554-6852-4A55-81BB-AC02C7A8CFED}"

Press any key to continue . . .
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**